### ISSUE TITLE
License?

### ISSUE THREAD

**Issue by M4a1x** [KITrobotics/Latex_Template#26]
Hey there,

I'm currently writing my thesis at KIT and was looking for a good template when I stumbled upon yours. I really like it! Just a quick question as I could not find a LICENSE.md or similar:

Can I use this for my thesis and if so should I mention it somewhere? What would be the best place?

Cheers,
Max

**Comment by destogl** [comment]
You can just use it :)

It is on the end created for our KIT students and from KIT scientific staff. We would appreciate to get any comments, bug reports and new features.

Cheers,
Denis

**Comment by M4a1x** [comment]
Hi Denis,

thanks for the heads up. I already started playing with it and found some small things (see #27).
IMHO would the repo benefit from an explicit license, as no [license usually means no permission](https://choosealicense.com/no-permission/)

As there's a specific license for LaTeX templates used by (afaik) basically all LaTeX packages (e.g. koma-script) I'd suggest to have a look at it: [The LaTeX Project Public License](https://www.latex-project.org/lppl/). 

Basically - what I gathered through google - the license says that everything compiled ('run') from the template can be distributed however that person wants (e.g. the pdf, the thesis, ..), but if the source itself is modified *and* distributed all changes must be disclosed, but it's not copyleft.

[Here's a small explanation on stackexchange by one of its authors on how it's applied](https://tex.stackexchange.com/a/82719) and a [Rundown by tldrlegal](https://tldrlegal.com/license/latex-project-public-license-v1.3c-(lppl-1.3c))

Just some food for thought. Thanks for the permission and the great work.

Cheers,
Max
